src/formats/yarle/process-resources.ts

```diff
@@ -77,9 +77,13 @@ const processResource = (workDir: string, resource: any): any => {
 	// Skip unknown type as we don't know how to handle
 	// Source: https://dev.evernote.com/doc/articles/data_structure.php
 	// "The default type "application/octet-stream" should be used if a more specific type is not known."
-	if (resource.mime === 'application/octet-stream') {
+	// Update: 
```

Comment:
@lishid or @ericaxu, does this seem reasonable to you? I'm not too familiar with Yarle.